tests/cpptests/redismock/redismock.cpp

```diff
@@ -819,7 +819,6 @@ static void registerApis() {
   REGISTER_API(FreeString);
   REGISTER_API(RetainString);
   REGISTER_API(HoldString);
-  REGISTER_API(TrimStringAllocation);
```

Comment:
I don't see a reason not to keep the mock implementation here for future usage/bug fixes

Reply:
It's redis 7 API, we could never use it in 2.6 anyway...